src/interfaces/ManagesImages.java

```diff
@@ -33,4 +33,13 @@
 	 * @param imageList
 	 */
 	public void setIndexImages(int[] imageList);
+
+	public void shuffleIndexImages(int[] imageList, int totalImages);
```

Comment:
Daca te uiti cu atentie la semnaturile metodelor din aceasta interfata o sa observi ca unul dintre parametrii se repeta. E vorba despre _int[] imageList_.

Acesta poate fi dat obiectului, la initializare, prin intermediul constructorului. Astfel, nu va mai fi nevoie ca parametru pentru fiecare metoda in parte.

De indata ce muti comentariul despre care spunem mai devreme, putem inchide si acest pull request, iar mai apoi se poate deschide altul cu modificarile legate de _int[] imageList_.

Reply:
Tot aici, mai poti sa citesti despre un alt principiu din Software Development. E vorba despre DRY (Don't Repeat Yourself).